test/api.test.ts

```diff
@@ -0,0 +1,233 @@
+// import request from "supertest";
+import { expect, assert } from "chai";
+import sinon from "sinon";
+import request from "supertest";
+import App from "../src/app";
+import { adapters, StorageService } from "../src/storage";
+import {
+  BidPayloadPostDto,
+  BidsAccountsDto,
+  BidsListDto,
+  BidPostDto,
+  VerifyBidResponse,
+  Auction,
+  BidsDto,
+} from "../src/types";
+import * as util from "../src/util";
+import * as auctions from "../src/util/auctions";
+import * as contracts from "../src/util/contracts";
+
+describe("Test API Endpoints", async () => {
+  sinon.stub(contracts, "encodeBid").returns(Promise.resolve(""));
+
+  const stubbedStorageService = {
```

Comment:
We should consider at some point using dependency injection and creating a simulated storage backend to avoid nasty stubbing like this.